packages/assets-controllers/src/AccountTrackerController.ts

```diff
@@ -157,6 +158,9 @@ export class AccountTrackerController extends BaseController<
     const accounts = { ...this.state.accounts };
     for (const address in accounts) {
       await safelyExecuteWithTimeout(async () => {
+        if (!this.ethQuery) {
+          throw new Error('Provider not set');
+        }
```

Comment:
This could be simplified like this. No strong opinion on this, though.
```suggestion
        assert(this.ethQuery, 'Provider not set.');
```